src/platform/windows/backtrace_windows.c

```diff
@@ -0,0 +1,143 @@
+#include "platform/backtrace.h"
+
+#include "SDL.h"
```

Comment:
External includes should be after internal includes (`SDL.h` should come after).

Reply:
Done